external-import/misp/src/misp.py

```diff
@@ -1316,16 +1316,17 @@ def prepare_elements(self, galaxies, tags, author, markings):
                     else:
                         aliases = [name]
                     if name not in added_names:
+                        x_mitre_id = None
+                        if "external_id" in galaxy_entity["meta"]:
+                            x_mitre_id = galaxy_entity["meta"]["external_id"][0]
                         elements["attack_patterns"].append(
                             AttackPattern(
                                 name=name,
                                 description=galaxy_entity["description"],
                                 created_by_ref=author,
                                 object_marking_refs=markings,
                                 custom_properties={
-                                    "x_mitre_id": galaxy_entity["meta"]["external_id"][
```

Comment:
Wasnt quite sure if I should set x_mitre_id to null or remove it from the dict

Reply:
Hey, Thanks for the PR. I know this is a bit of of a nitpicking, but your solution is only a partial fix. 
In case the `external_id` list contains no elements, another unhandled exception is thrown. 
```
> galaxy_entity["meta"]["external_id"] = []
> galaxy_entity["meta"]["external_id"][0]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
IndexError: list index out of range
```
Could you please improve this?

Reply:
Sure thing, pushing this (below) now,  Not sure how we want to handle multiples or if we even do tho

```python
x_mitre_id = None
if "external_id" in galaxy_entity["meta"]:
    if len(galaxy_entity["meta"]["external_id"]) > 0:
        x_mitre_id = galaxy_entity["meta"]["external_id"][0]
```